CHANGES.rst

```diff
@@ -4,6 +4,8 @@
 Enhancements and Fixes
 ----------------------
 
+- Fix ``pyvo.registry.Author`` that missed a table join [#515]
```

Comment:
I feel this can be rephrased a bit as it would be more informative for the users to know what it fixed rather than what the underlying bug was.
```suggestion
- Fix ``pyvo.registry.Author`` to allow registry searches with author constraints. [#515]
```